src/lib/server/events/gcal.ts

```diff
@@ -0,0 +1,40 @@
+import { Temporal } from '@js-temporal/polyfill';
+import type { ClubEvent } from '$lib/public/events/event';
+import { google } from 'googleapis';
```

Comment:
This import statement.

```ts
import { google } from 'googleapis';
```

It imports a monolith housing every Google API ever generated. We only need the Google Calendar API, so let's utilize the technique of [_tree shaking_](https://developer.mozilla.org/en-US/docs/Glossary/Tree_shaking) to skip importing unused Google APIs (related: <https://github.com/googleapis/google-api-nodejs-client/issues/806#issuecomment-631058329>, <https://github.com/googleapis/google-api-nodejs-client/pull/2557>).

```suggestion
import { calendar } from '@googleapis/calendar';
```